.agent/workflows/develop.md

```
@@ -3,206 +3,106 @@ description: Implement requirements from a clarified document using DiSCOS, AGEN
 ---
 
 > **Pipeline**: `/clarify` → `/answer` → `/develop` (3/3) · [Status Lifecycle](./_shared/status-lifecycle.md)
->
-> **Estimated context: ~2.0K tokens** (this workflow)
+> **Estimated context: ~1.4K tokens**
 
 ---
 
 ## 1. Resolve Input
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)